apps/showcase/src/assets/trainings/sdk/program.json

```diff
@@ -0,0 +1,130 @@
+[
```

Comment:
Should avoid array on root for JSON, even if it would be a single field.
it would enable the support of `$chema` when you will provide it.
(I think all structured JSON we define shoul have their JsonSchema associated)